caproto/server/records.py

```diff
@@ -122,7 +123,7 @@ class RecordFieldGroup(PVGroup):
     new_alarm_status = pvproperty(
         name='NSTA',
         dtype=ChannelType.ENUM,
-        enum_strings=menus.menuAlarmStat.get_string_tuple(),
+        enum_strings=menus.menuAlarmStat.get_string_tuple()[:MAX_ENUM_STATES],
```

Comment:
It seems like it could be easy to forget to add this. Should it be baked into `get_string_tuple()`? Or exposed as an option to that method?

Reply:
That sounds like a reasonable solution, @danielballan.

Reply:
Addressed in 2234905